security/security-core/src/main/java/io/camunda/security/auth/OidcGroupsLoader.java

```diff
@@ -47,24 +50,28 @@ public List<String> load(final Map<String, Object> claims) {
     if (groupsClaim == null) {
       return null;
     }
-    final var claimGroups = JsonPath.using(CONFIGURATION).parse(claims).read(groupsClaim);
     final var groups = new ArrayList<String>();
-    if (claimGroups != null) {
-      if (claimGroups instanceof final String group) {
-        groups.add(group);
-      } else if (claimGroups instanceof final List<?> list) {
-        for (final Object o : list) {
-          if (o instanceof final String group) {
-            groups.add(group);
-          } else {
-            throw new IllegalArgumentException(
-                String.format(DERIVED_GROUPS_ARE_NOT_STRING_ARRAY, groupsClaim));
+    try {
+      final var claimGroups = JsonPath.using(CONFIGURATION).parse(claims).read(groupsClaim);
+      if (claimGroups != null) {
+        if (claimGroups instanceof final String group) {
+          groups.add(group);
+        } else if (claimGroups instanceof final List<?> list) {
+          for (final Object o : list) {
+            if (o instanceof final String group) {
+              groups.add(group);
+            } else {
+              throw new IllegalArgumentException(
+                  String.format(DERIVED_GROUPS_ARE_NOT_STRING_ARRAY, groupsClaim));
+            }
           }
+        } else {
+          throw new IllegalArgumentException(
+              String.format(DERIVED_GROUPS_ARE_NOT_STRING_ARRAY, groupsClaim));
         }
-      } else {
-        throw new IllegalArgumentException(
-            String.format(DERIVED_GROUPS_ARE_NOT_STRING_ARRAY, groupsClaim));
       }
+    } catch (final PathNotFoundException exception) {
+      LOGGER.warn("Could not load groups from claims {}", claims, exception);
```

Comment:
❓ Sorry I miss some context here. When is this exception thrown? Is it a misconfiguration on the user's side? I'm wondering if the log level should be an error or a warning mostly.